x-pack/test/functional/es_archives/filebeat/threat_intel/data.json

```diff
@@ -0,0 +1,276 @@
+        "ingested": "2021-01-26T11:09:06.616763Z",
+        "kind": "enrichment",
+        "module": "threatintel",
+        "reference": "https://urlhaus.abuse.ch/url/978782/",
+        "type": "indicator"
+      },
+      "fileset": {
+        "name": "abuseurl"
+      },
+      "input": {
+        "type": "httpjson"
+      },
+      "service": {
+        "type": "threatintel"
+      },
+      "tags": [
+        "threatintel-abuseurls",
+        "forwarded"
+      ],
+      "threat": {
+        "indicator": {
+          "description": "this should match auditbeat/hosts on both port and ip",
+          "ip": "45.115.45.3",
+          "port": 57324,
+          "first_seen": "2021-01-26T11:06:03.000Z",
+          "provider": "geenensp",
+          "type": "url"
+        }
+      },
+      "threatintel": {
+        "abuseurl": {
+          "blacklists": {
+            "spamhaus_dbl": "not listed",
+            "surbl": "not listed"
+          },
+          "larted": true,
+          "tags": [
+            "32-bit",
+            "elf",
+            "mips"
+          ],
+          "threat": "malware_download",
+          "url_status": "online"
+        }
+      }
+    }
+  }
+}
+
+{
+  "type": "doc",
+  "value": {
+    "id": "978787",
+    "index": "filebeat-8.0.0-2021.01.26-000001",
+    "source": {
+      "@timestamp": "2021-01-26T11:09:05.529Z",
+      "agent": {
+        "ephemeral_id": "b7b56c3e-1f27-4c69-96f4-aa9ca47888d0",
+        "id": "69acb5f0-1e79-4cfe-a4dc-e0dbf229ff51",
+        "name": "MacBook-Pro-de-Gloria.local",
+        "type": "filebeat",
+        "version": "8.0.0"
+      },
+      "ecs": {
+        "version": "1.6.0"
+      },
+      "event": {
+        "category": "threat",
+        "created": "2021-01-26T11:09:05.529Z",
+        "dataset": "threatintel.abuseurl",
+        "ingested": "2021-01-26T11:09:06.616763Z",
+        "kind": "enrichment",
+        "module": "threatintel",
+        "reference": "https://urlhaus.abuse.ch/url/978782/",
+        "type": "indicator"
+      },
+      "fileset": {
+        "name": "abuseurl"
+      },
+      "input": {
+        "type": "httpjson"
+      },
+      "service": {
+        "type": "threatintel"
+      },
+      "tags": [
+        "threatintel-abuseurls",
+        "forwarded"
+      ],
+      "threat": {
+        "indicator": {
+          "description": "this should match auditbeat/hosts on ip",
+          "ip": "45.115.45.3",
+          "first_seen": "2021-01-26T11:06:03.000Z",
+          "provider": "other_provider",
+          "type": "ip"
+        }
+      },
+      "threatintel": {
+        "abuseurl": {
+          "blacklists": {
+            "spamhaus_dbl": "not listed",
+            "surbl": "not listed"
+          },
+          "larted": true,
+          "tags": [
+            "32-bit",
+            "elf",
+            "mips"
+          ],
+          "threat": "malware_download",
+          "url_status": "online"
+        }
+      }
+    }
+  }
+}
```

Comment:
As talked about over video conference let's see if we can trim this and the mappings down to their essence and only have what we absolutely need in these for testing.

This will make maintainers lives easier and make things easier to understand. If someone needs to test other portions or parts of threat intelligence we should encourage them to create and use different index mappings and data sets specific to those tests and avoid mixing extra baggage between tests.

It seems to be the better way for this to all work.

Reply:
Agreed; I will trim down this file, and also create a smaller auditbeat index containing just the few events that we care about here (plus 1-2 extras to ensure no false positives). In a followup PR 😜